dmoj/checkers/linecount.py

```diff
@@ -28,7 +28,8 @@ def check(
             count += 1
 
     return CheckerResult(
-        count == len(judge_lines), point_value * (1.0 * count / len(judge_lines)), ''.join(cases) if feedback else ""
+        count == len(judge_lines), point_value * (1.0 * count / len(judge_lines)),
+        extended_feedback='Case Feedback:\n' + '\n'.join(cases) if feedback else ''
```

Comment:
`''.join` might be better so it's not one giant column going all the way down.

@Xyene any thoughts?

Reply:
Not Xyene, but if my memory serves me right then `''.join` is what MCPT does and it looks quite nice. I think the giant column wouldn't look nice either.